### PR TITLE
Pitbulls Are Friendly

### PR DESCRIPTION
**Pull request by ShatteredSwords**
# Description

This PR makes Pitbulls SimpleNeutral faction, and also, gets rid of their aggresiveness, 

---

# Changelog

:cl:
- tweak: Pitbulls are now neutral to all crew.

### CODE REVIEW

**Comment by ShatteredSwords** [comment]
gimme a moment I'm both playing the game and working on this

**Comment by ShatteredSwords** [comment]
Alright it's ready.

File: Resources/Prototypes/Entities/Mobs/NPCs/dogs.yml [hunk 1]
```diff
@@ -69,13 +69,7 @@
   - type: DogVision
   - type: NpcFactionMember
     factions:
-    - Pibble
-  - type: HTN
-    rootTask:
-      task: SimpleHostileCompound
-    blackboard:
-      NavSmash: !type:Bool
-        true
```
Comment (VMSolidus) [inline Resources/Prototypes/Entities/Mobs/NPCs/dogs.yml:78]: Don't remove lines, simply comment them out.

**Review by VMSolidus — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Mobs/NPCs/dogs.yml (hunk 1 above)
Comment (ShatteredSwords) [reply Resources/Prototypes/Entities/Mobs/NPCs/dogs.yml:78]: You right

**Review by Memeji — APPROVED** [review]

**Review by Fansana — APPROVED** [review]

**Pull request merged**